Mathlib/Analysis/Complex/Arg.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2022 Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+
+! This file was ported from Lean 3 source module analysis.complex.arg
+! leanprover-community/mathlib commit 17ef379e997badd73e5eabb4d38f11919ab3c4b3
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.Analysis.SpecialFunctions.Complex.Arg
+
+/-!
+# Rays in the complex numbers
+
+This file links the definition `SameRay ℝ x y` with the equality of arguments of complex numbers,
+the usual way this is considered.
+
+## Main statements
+
+* `Complex.sameRay_iff` : Two complex numbers are on the same ray iff one of them is zero, or they
+  have the same argument.
```

Comment:
@ericrbg can you have a look at the doc? The corrent mathlib3 doc is inaccurate, and I just removed the wrong line here. Feel free to add a correct one. Thanks!

Reply:
I guess the Lean3 doc says the `Iff.Neg` version, kind of, haha. I'll push a fix in both places.